app/helpers/guide_helper.rb

```diff
@@ -36,10 +36,12 @@ def guide_community_options_for_select
 
   def topic_section_options_for_select
     Topic.includes(:topic_sections).map do |topic|
+      relative_path = topic.path.gsub("/service-manual", "")
```

Comment:
Thanks - I've gone for the `sub` version as it'll hopefully be easier to use when we go around moving all the "/service-manual" strings we have lurking into config.